### DISCUSSION TITLE
nest.js build: type error

### DISCUSSION THREAD

**Discussion by zenitlab** [gilbarbara/react-inlinesvg#191] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/37064289/180429551-5bee5da1-563d-4eb8-82dc-67dfeebd64a5.png)

nest.js build: type error, how to fix it?

**Comment by gilbarbara** [comment, marked as answer]
Did you check [this](https://github.com/gilbarbara/react-inlinesvg/discussions/186)?

**Reply by zenitlab** [reply]
yes，I've solved this problem，tks
